markdownlint.js

```diff
@@ -38,7 +35,7 @@ const processCwd = process.cwd();
 
 function readConfiguration(userConfigFile) {
   const jsConfigFile = /\.js$/i.test(userConfigFile);
-  const rcArgv = minimist(process.argv.slice(2));
```

Comment:
I was worried about this change, but it looks like the result wasn't used. There is no third parameter to run-con, could you remove it a few lines below, please?

https://www.npmjs.com/package/run-con

Reply:
Good catch.